Mlem/App/Views/Root/Tabs/Settings/AccountSignInSettingsView.swift

```diff
@@ -23,7 +24,13 @@ struct AccountSignInSettingsView: View {
                     }
                 }
             }
+            Section {
+                Button("Change Password", systemImage: "key") {
```

Comment:
*nit*

```suggestion
                Button("Change Password", systemImage: Icons.security) {
```

(this also needs to be added to `Icons`, and the "Sign in & Security" settings page should be updated to use `Icons.securityFill`)